Kwf/Controller/Action/Auto/Import.php

```diff
@@ -30,7 +30,8 @@ protected function _initFields()
 
     protected function _beforeSave(Kwf_Model_Row_Interface $row)
     {
-        $model = Kwf_Model_Abstract::getInstance('Kwf_Uploads_Model');
+        $uploadsModelClass = Kwf_Config::getValue('uploadsModelClass');
```

Comment:
Das könnte man doch in eine statische Funktion im Uploads-Model machen. und überall wo das UploadsModel verwendet wird verwendet man anstelle dessen Kwf_Uploads_Model::getInstance() (wobei da vermutlich ein Konflikt mit dem getInstance vom Kwf_Model_Abstract sein wird. Also vl nicht getInstance() nennen, sondern anders.